### ISSUE TITLE
pi0.5 on LIBERO simulation reproduce, zero success rate on pi05_libero_base

### ISSUE THREAD

**Issue by zhuxiaofan117** [huggingface/lerobot#2533]
Hi all, 
I am very new to Lerobot right now and I want to start with reproducing pi0.5 results on libero bench.

I was a bit puzzled by the difference of pi05_base, pi05_libero_base and pi05_libero_finetuned. Because I had the evaluation results as follows:

| libero_tasks | libero_spatial | libero_object | libero_goal | libero_10 |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| pi05_base | 0.0 | 0.0 | 0.07 | 0.0 |
| pi05_libero_base | 0.0 | 0.0 | 0.00 | 0.0 |
| pi05_libero_finetuned | 0.98 | 1.0 | 0.98 | 0.96 |

According to [this](https://huggingface.co/lerobot/pi05_libero_base), pi05_libero_base is a pytorch conversion of original openai's pi05_libero, which reports a 96.85% average SR(success rate) on libero bench. But my eval of pi05_libero_base on lerobot-libero bench returns zero SR, which made me guess what's going on in my reproduction.

My commands for eval are like(take evaluating pi05_libero_base for example):

```
lerobot-eval \
   --env.type=libero \
   --env.task=libero_spatial,libero_object,libero_goal,libero_10 \
   --eval.batch_size=1 \
   --eval.n_episodes=10 \
   --policy.path={$my_huggingface_cache_dir}/lerobot/pi05_libero_base/ \
   --policy.n_action_steps=10 \
   --output_dir=./eval_logs/test5_pi05_libero_base \
   --env.max_parallel_tasks=1 \
```

If anyone has any thoughts, please drops a comment. Thanks in advance.

**Comment by pkooij** [comment]
We have a different pipeline than Physical intelligence, this means that although the weights are the same porting their pipeline: image processing and normalization to our pipeline is not very straight forward. That's why we finetuned the weights instead of using their checkpoint out of the box.

**Comment by flowmatching** [comment]
> We have a different pipeline than Physical intelligence, this means that although the weights are the same porting their pipeline: image processing and normalization to our pipeline is not very straight forward. That's why we finetuned the weights instead of using their checkpoint out of the box.

I kindly request some clarification on how the pi05_libero_finetuned weights were obtained. Could you confirm if the process was as follows?

Conversion: OpenPi JAX weights were directly converted to pi05_libero_base.

Fine-tuning: Continued SFT was then applied to the base model to produce pi05_libero_finetuned.

**Comment by lucasjinreal** [comment]
The lerobot pi0 pi0.5 version has a relatively too poor performance compare with openpi version.

**Comment by zhangpf21** [comment]
Excuse me, I encountered the same problem. Have you solved it?

**Comment by jashshah999** [comment]
For anyone trying to reproduce results and getting inconsistent numbers, [lerobot-bench](https://github.com/jashshah999/lerobot-bench) automates the multi-seed evaluation and tells you whether differences are statistically significant. Useful when you are not sure if 0% success is a real problem vs bad luck on a single seed.

https://github.com/jashshah999/lerobot-bench
